blib2to3/pgen2/driver.py

```diff
@@ -128,7 +128,10 @@ def parse_stream(self, stream: IO[Text], debug: bool = False) -> NL:
         return self.parse_stream_raw(stream, debug)
 
     def parse_file(
-        self, filename: Path, encoding: Optional[Text] = None, debug: bool = False,
+        self,
```

Comment:
This should be formatted as it was before your change.

Reply:
I found that if line is a bit longer, we get

```python
self, filename: Path, encoding: Optional[
    Text
] = None, debug: bool = False,
```

Is it okay? Can easily be fixed, I just should remove `[delimiter_split, standalone_comment_split]` for LHS values in brackets.

Reply:
Removed trailing comma, now this line is the same as it was.